datasets/dev/data.sql

```diff
@@ -317,5 +317,5 @@ Yellow
+Blue','','','','','What is your favorite color?','What''s up?','What''s your favorite day of the week?','','','Pick-list','Text area','Text box required','','','True','','Do not ask','Do not ask','Do not ask','True','Do not ask','Ask','Ask with type require','Do not ask','Do not ask','Ask','Ask but do not require','Do not ask','Parent/Guardian;Other','General Public','','0.0','Don''t ask','','Full matching contact creation with duplicate management','Use Salesforce Duplicate Rule','','2020-10-18','Body text for the appointment/options page explains what these appointments are about.','This is the header of the appointment/options page.','What does it mean to cancel a event.','Online Cancel of Registration Heading','The event has been cancelled.','Explain the registration has been received from the client at this point and is complete','Registration has been received title.','This footer appears on every event registration page in the footer.','Explain here that the event has reached capacity and is closed.','Event Home','','Test Event','This description appears in feed and should be concise','','Active','Description of submission being the final act in the registration play.','Heading for the Submit Page','Admissions Event','<p><strong>Rich Text</strong> description of the event that appears on the first registration page.</p>','Register','Touchpoint','False','False','123 Evergreen Blvd., St. Paul, MN','https://g.page/sociablecider?share','The best place in the World','','5.0','Test Event','False','','','','No limit','2020-08-18','GeneralSLDS','','','','','');
```

Comment:
@Scarrien ,

When I tried to build this branch fresh I got the following error on the data load:

> When using a Salesforce Duplicate Rule for matching, the Custom Metadata Contact Matching Method and Contact Creation Duplicate Rules are required

Perhaps we just need to not set this option 'Use Salesforce Duplicate Rule' until duplicate rules are in place. We probably don't need to set anything for 'Full matching contact creation with duplicate management' anymore either?

I hope slimming down this code base is a relief without all the old St. Thomas stuff mucking it up. Great work on this!

I also rebuilt the object metadata in this request because of that error in sfdx where it obliterates the meta data on dx_pull.

Thanks!
Thad